src/s1reader/s1_annotation.py

```diff
@@ -515,6 +579,167 @@ def load_from_zip_file(cls, path_aux_cal_zip: str, pol: str, str_swath: str):
 
         return cls
 
+
+@dataclass
+class SwathRFIInfo:
```

Comment:
Sorry for the "back and fourth" regarding these acronyms camel case convention, but since we decided to use "Rfi" in `dict_datatype_rfi`, I think we should also revert the changes to the class name here from `SwathRFIInfo` to `SwathRfiInfo`.

Reply:
Makes sense. We should stick to the one convention.